rr_sequencer.py

```diff
@@ -343,6 +343,10 @@ def perform_action(self):
             # Unmark targeted regions
             if 'key' in self.options:
                 self.clear_highlights(self.options['key'].strip())
+        elif self.action == 'select':
+            self.view.sel().clear()
+            for region in self.replace_obj.target_regions:
```

Comment:
I believe you can just use:

```
self.view.sel().add_all(self.replace_obj.target_regions)
```

This will save from refreshing and generating events between every selection that gets added individually.  Generally it should be quicker as on_selection_modified events won't be triggered for every selection added.